genie-core/src/main/java/com/netflix/genie/core/jobs/workflow/impl/GenieBaseTask.java

```diff
@@ -41,6 +42,13 @@
 @Slf4j
 public abstract class GenieBaseTask implements WorkflowTask {
 
+    protected static final String NO_ID_FOUND = "<no id>";
```

Comment:
This was duped in a few places, now it's here and used by various tasks subclasses